### PR TITLE
Added zenodo badge/links to readme

### PR DESCRIPTION
**Pull request by chris-cheshire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![GitHub Actions CI Status](https://github.com/nf-core/cutandrun/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/cutandrun/actions?query=workflow%3A%22nf-core+CI%22)
 [![GitHub Actions Linting Status](https://github.com/nf-core/cutandrun/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/cutandrun/actions?query=workflow%3A%22nf-core+linting%22)
 [![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/cutandrun/results)
-[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.XXXXXXX)
+[![DOI](http://img.shields.io/badge/DOI-10.5281/zenodo.5653536-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.5653536)
```
Comment (drpatelh) [inline README.md:6]:
```suggestion
[![DOI](http://img.shields.io/badge/DOI-10.5281/zenodo.5653535-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.5653535)
```

File: README.md [hunk 2]
```diff
@@ -92,8 +92,7 @@ For further information or help, don't hesitate to get in touch on the [Slack `#
 
 ## Citations
 
-<!-- TODO nf-core: Add citation for pipeline after first release. Uncomment lines below and update Zenodo doi and badge at the top of this file. -->
-<!-- If you use  nf-core/cutandrun for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX) -->
+If you use  nf-core/cutandrun for your analysis, please cite it using the following doi: [10.5281/zenodo.5653536](https://doi.org/10.5281/zenodo.5653536)
```
Comment (drpatelh) [inline README.md:95]:
```suggestion
If you use nf-core/cutandrun for your analysis, please cite it using the following doi: [10.5281/zenodo.5653535](https://doi.org/10.5281/zenodo.5653535)
```

**Review by drpatelh — COMMENTED** [review]
Need to update [this line](https://github.com/nf-core/rnaseq/blob/964425e3fd8bfc3dc7bce43279a98d17a874d3f7/lib/WorkflowMain.groovy#L13) too.

File: README.md (hunk 1 above)
Comment (drpatelh) [reply README.md:6]: The version we want to use is the one that says "Cite all versions" so we don't have to update the Zenodo DOI after every release.

Comment (chris-cheshire) [reply README.md:6]: How come this changed to a 5 instead of 6 at the end?

File: README.md (hunk 2 above)
Comment (chris-cheshire) [reply README.md:95]: Same here

File: README.md (hunk 1 above)
Comment (ewels) [reply README.md:6]: Because of the all versions thing..

**Review by ewels — APPROVED** [review]
LGTM after the all versions change

**Review by drpatelh — APPROVED** [review]

**Pull request merged**